src/System.Devices.Gpio/System.Devices.Gpio.Samples/Bme280.cs

```diff
@@ -0,0 +1,419 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime.CompilerServices;
+
+namespace System.Devices.Gpio.Samples
```

Comment:
For a sample, this is missing A LOT of comments ☹️ 

Could you draw inspiration from https://github.com/Microsoft/Windows-universal-samples/blob/master/Samples/IoT-I2C/cs/Scenario1_ReadData.xaml.cs#L42-L63 ?